src/main/java/core/net/MyConnector.java

```diff
@@ -579,10 +579,9 @@ public String fetchRegionID(int teamId) {
 	}
 
 
-	public String fetchTeamDetails(int teamId)
-	{
+	public String fetchTeamDetails(int teamId) {
```

Comment:
@tychobrailleur do we need both methods get- and fetchTeamDetails? Is there a chance to handle it with one of them?

Reply:
Wow, well spotted!  I have collapsed the two, now.